src/libraries/System.Net.Security/src/System/Net/Security/SslConnectionInfo.cs

```diff
@@ -21,5 +21,8 @@ internal partial struct SslConnectionInfo
         public int KeyExchKeySize { get; private set; }
 
         public byte[]? ApplicationProtocol { get; internal set; }
+#if DEBUG
+        public bool TlsResumed { get; private set; }
+#endif
```

Comment:
Is there an upside to leaving this in?

Reply:
I was more questioning if it is useful to have debug-only fields. Since most crash-dumps or bug reports come from users who have Release bits, this property will not be there.

So the only case where this code will be useful is when we attempt to reproduce those in Debug builds, but since we can already detect TlsResume from packet captures, this does not seem that useful to me....

Reply:
right. This is there only so we can write tests for it and query it programatically. If you have other ideas how to test the resume control please let me know. 

I would also like to write test for https://github.com/dotnet/runtime/issues/75434 to make sure resume works as there is significant performance impact.

Reply:
But then, the test needs to be also Debug only, do we already have anything similar in runtime?

I guess I am not that opposed to it anymore.

Reply:
yes, some of the test that depend on reflection are guarded by `#if DEBUG`